### ISSUE TITLE
validation loss always comes as nan

### ISSUE THREAD

**Issue by sgkcfc** [PacktPublishing/Hands-On-Deep-Learning-for-Finance#11]
The validation loss is always coming out to be nan no matter what. Also looks like the validation loop is nested inside the training loop. not sure if that is the cause of the problem. Nan is the output for all the epochs

Epoch: 79/100 Iteration: 80 Validation loss:    nan Validation acc: nan

the validation data sets are not null as i checked them. 

if not a bug can a reference .hd5 be provided.

**Comment by sgkcfc** [comment]
The reason for this is that batch size is always greater than the number of validation datasets. I think this is accomplished in tensorflow_Setup file where the split is formed and stored in the database file

**Comment by abhandari73** [comment]
Please refer to the codes in Chap06 directory.
